src/gromacs/nblib/tests/topology.cpp

```diff
@@ -180,6 +180,19 @@ TEST(NBlibTest, TopologyHasExclusions)
     compareLists(testExclusions, refExclusions);
 }
 
+TEST(NBlibTest, TopologyHasSequencing)
```

Comment:
We should also have a test with EXPECT_THROW where we use various invalid data such as wrong indices.

Reply:
Yes, I'm addressing this in the TopoBonds commit.

Reply:
PR I mean